src/textures/Texture.js

```diff
@@ -304,5 +304,6 @@ class Texture extends EventDispatcher {
 
 Texture.DEFAULT_IMAGE = null;
 Texture.DEFAULT_MAPPING = UVMapping;
+Texture.DEFAULT_ANISOTROPY = 1;
```

Comment:
Hm, why the case differs between `Object3D.DefaultUp`/`Object3D.DefaultMatrixAutoUpdate` and `Texture.DEFAULT_IMAGE`/`Texture.DEFAULT_MAPPING`/`Texture.DEFAULT_ANISOTROPY`?

Reply:
I'm not sure but I think @mrdoob lately stated his preference for this nomenclature. Unfortunately, I don't find the discussion right now.

For the time being, it's sufficient when we adhere to naming conventions per file. At a later point, we can unify the syntax.

Reply:
It probably should be `Object3D.DEFAULT_UP` and `Object3D.DEFAULT_MATRIXAUTOUPDATE`...

@LeviPesin Are these the only ones that would need to be changed?

Reply:
I can make these changes for consistency if @mrdoob approves.
```js
// from
Object3D.DefaultUp = /*@__PURE__*/ new Vector3( 0, 1, 0 );
Object3D.DefaultMatrixAutoUpdate = true;
Object3D.DefaultMatrixWorldAutoUpdate = true;

// to
Object3D.DEFAULT_UP = /*@__PURE__*/ new Vector3( 0, 1, 0 );
Object3D.DEFAULT_MATRIXAUTOUPDATE = true;
Object3D.DEFAULT_MATRIXWORLDAUTOUPDATE = true;

// from
Euler.DefaultOrder = 'XYZ';
Euler.RotationOrders = [ 'XYZ', 'YZX', 'ZXY', 'XZY', 'YXZ', 'ZYX' ];

// to
Euler.DEFAULT_ORDER = 'XYZ';
Euler.RotationOrders // wait. why is this here? leave as-is -- or deprecate it?

// as-is
Texture.DEFAULT_IMAGE = null;
Texture.DEFAULT_MAPPING = UVMapping;
Texture.DEFAULT_ANISOTROPY = 1;
```

Reply:
> Euler.RotationOrders // wait. why is this here? leave as-is -- or deprecate it?

I vote to remove it. Possible euler rotation orders should be clear without defining them in an array.

Reply:
Completely remove it with an added migration note -- or add a 10-month deprecation warning and retain it?